sycl/source/detail/device_info.hpp

```diff
@@ -279,6 +280,25 @@ struct get_device_info<std::vector<info::execution_capability>,
   }
 };
 
+// Specialization for built in kernel identifiers
+template <>
+struct get_device_info<std::vector<kernel_id>,
+                       info::device::built_in_kernel_ids> {
+  static std::vector<kernel_id> get(RT::PiDevice dev, const plugin &Plugin) {
+    std::string result =
+        get_device_info<std::string, info::device::built_in_kernels>::get(
```

Comment:
I think we will get the deprecation warning for info::device::built_in_kernels here.

Reply:
Since this is not a user-visible include file but is in `sycl/source`, our compiler should never be compiling this code, right?

Reply:
It is better to avoid warnings in RT library builds as well.